papis/commands/add.py

```diff
@@ -364,6 +357,16 @@ def run(paths: List[str],
         confirm = True
         has_duplicate = True
 
+    papis.database.get().add(tmp_document)
```

Comment:
Not a big fan of adding a temporary document to the library. Does this even get cleaned up?

Reply:
I went with db.add(doc) because it seemed like a safer, higher-level entry point. Indeed, it is probably better to use maybe_compute_id directly, as you do above.

Reply:
In fact, I just relocated the db.add call without realizing it would add the temporary path to the cache, I just reverted that commit.